name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/NIBCBankPDFExtractor.java

```diff
@@ -145,12 +145,20 @@ private void addDividendeTransaction()
                     t.setShares(asShares(v.get("shares")));
                     t.setSecurity(getOrCreateSecurity(v));
                 })
+                
+                // Zahlbarkeitstag 08.06.2021 Dividende pro Stück 5,00 PLN
+                .section("date").match("^Zahlbarkeitstag (?<date>\\d+.\\d+.\\d{4}).*")
+                .assign((t,v) -> t.setDateTime(asDate(v.get("date"))))
+                
+                // Ex-Tag 12.12.2019 fonds) 0,208116300 USD
+                .section("exDate").match("^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}).*$")
```

Comment:
Fix formatting with linebreak the source
maybe change regEx to
`^Ex-Tag (?<exDate>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) .*$`

Reply:
changed